less/index.less

```diff
@@ -50,10 +50,267 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+  height: 100%;
 }
 
+// Variables
+@fontFamily: 'Lato', sans-serif;
+@fontHeading: 'Roboto Slab', serif;
+@defaultFontSize: 1.6rem;
+@defaultFontColor: #1d1d1d;
+@accentColor: #ffcf23;
+@borderColor: #dedede;
+@spaceBottom: 20px;
+@tabletWidth: 800px;
+@mobileWidth: 500px;
+
+// Mixins
+.center() {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.customLayout(@jusitfy) {
+	display: flex;
+	justify-content: @jusitfy;
+}
+
+// Styling starts here
+html {
+	font-family: @fontFamily;
+	font-size: 62.5%;
+	color: @defaultFontColor;
+}
+
+p, li {
+	line-height: 1.5;
+	font-weight: 300;
+}
+
+h1, h2, h3, h4, h5, h6 {
+	font-family: @fontHeading;
+}
+
+strong {
+	font-weight: 900;
+}
+
+.container {
+	font-size: @defaultFontSize;
+	max-width: 1024px;
+	margin: 0 auto;
+}
+
+// Header
+header {
+	.main-nav {
+		width: 100%;
+		.center;
+
+		@media only screen and (max-width: @mobileWidth) {
+			flex-direction: column;
+		}
+
+		a {
+			margin: 10px 0;
+			padding: 10px 5%;
+			text-decoration: none;
+			color: #000;
+			font-weight: 400;
+
+			@media only screen and (max-width: @mobileWidth) {
```

Comment:
Nice job nesting in your viewports here to change the css of the specific elements. 💯